bindata/v3.11.0/kube-controller-manager/kubeconfig-cm.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: controller-manager-kubeconfig
+  namespace: openshift-kube-controller-manager
+data:
+  kubeconfig: |
+    apiVersion: v1
+    clusters:
+      - cluster:
+          certificate-authority: /etc/kubernetes/static-pod-resources/configmaps/serviceaccount-ca/ca-bundle.crt
+          server: https://localhost:6443
```

Comment:
why localhost?

Reply:
> why localhost?

we are on the hostnetwork and we cannot use the service network. it's this or individual master IPs.  this is easier for my head to understand.